### PR TITLE
Add known issue about overflow to docs

### PR DESCRIPTION
**Pull request by ed-xmos**
[AP-183](https://xmosjira.atlassian.net/browse/AP-183)

[AP-183]: https://xmosjira.atlassian.net/browse/AP-183?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by shuchitak — APPROVED** [review]

File: README.rst [hunk 1]
```diff
@@ -60,6 +60,12 @@ There are three different component options that support fixed factor of 3 up/do
  #. If voice quality (65 dB SNR) is required running on xCORE-200, use ds3_voice or us3_voice.
  #. If voice quality (75 dB SNR) is required running xcore-ai, use ff3_96t_ds or ff3_96t_us.
 
+.. warning::
+    Synchronous fixed factor of 3 and 3/2 downsample and oversample functions for voice applications optimized for the XS3 Vector Processing Unit
+    currently overflow rather than saturate in cases where a full scale input causes a perturbation above full scale at the output.
+    To avoid this scenario, please ensure that the input amplitude is always 3.5dB below full scale.
```
Comment (mbanth) [inline README.rst:66]: Please add a space between `3.5` and `dB`.  In general we should include a space between numbers and their units in our documentation.

File: doc/programming_guide/fixed_ratio_src.rst [hunk 2]
```diff
@@ -121,6 +121,12 @@ They have been designed for voice applications and, in particular, conformance t
 .. note::
     These filters will only run on XCORE-AI due to the inner dot product calculation employing the XS3 VPU.
 
+.. warning::
+    Synchronous fixed factor of 3 and 3/2 downsample and oversample functions for voice applications optimized for the XS3 Vector Processing Unit
+    currently overflow rather than saturate in cases where a full scale input causes a perturbation above full scale at the output.
+    To avoid this scenario, please ensure that the input amplitude is always 3.5dB below full scale.
```
Comment (mbanth) [inline doc/programming_guide/fixed_ratio_src.rst:127]: Please add a space between `3.5` and 'dB`.

File: README.rst (hunk 1 above)
Comment (ed-xmos) [reply README.rst]: Done

**Review by mbanth — APPROVED** [review]
Looks good.

**Pull request merged**